test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -199,6 +225,17 @@
                 body (parse-response resp)]
             (is (= 403 (get body "status")))))))))
 
+(deftest metrics-service-with-tk-auth
+  (testing "tk-auth works when included in bootstrap"
+    (with-app-with-config
+      app
+      (conj services authorization-service/authorization-service)
+      (merge metrics-service-config auth-config ssl-webserver-config)
+      (let [resp (http-client/get "https://localhost:8180/metrics/v2/list" ssl-opts)]
+        (is (= 200 (:status resp))))
+      (let [resp (http-client/get "https://localhost:8180/metrics/v2" ssl-opts)]
+        (is (= 403 (:status resp)))))))
```

Comment:
Can you explain why this one isn't supposed to work?

Reply:
Because it says "deny" in the auth.conf for that path.

Reply:
We actually have really good docs for tk-auth, IMO. This might help https://github.com/puppetlabs/trapperkeeper-authorization/blob/master/doc/authorization-config.md.

Reply:
because it's set to deny in the auth config above

Reply:
lol that's what I get for skimming, thanks